internal/host/statemachine.go

```diff
@@ -176,7 +176,9 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 	sm.AddTransition(stateswitch.TransitionRule{
 		TransitionType: TransitionTypeResettingPendingUserAction,
 		SourceStates: []stateswitch.State{
-			HostStatusResetting,
+			stateswitch.State(models.HostStatusResetting),
+			stateswitch.State(models.HostStatusDiscovering),
+			stateswitch.State(models.HostStatusKnown),
```

Comment:
I don't understand this change

Reply:
This is required since if one of the hosts is stuck in resetting we move all hosts to HostStatusResettingPendingUserAction

Reply:
Why?

Reply:
For the same reason we change all statuses to HostStatusResettingPendingUserAction when one host is after reboot stage, no?